compiler/src/dotty/tools/dotc/cc/CaptureSet.scala

```diff
@@ -1258,6 +1267,29 @@ object CaptureSet:
       res.myTrace = cs1 :: this.myTrace
       res
 
+    def description(using Context): String = cs match
+      case cs: Var =>
+        if !cs.levelOK(elem) then
+          val levelStr = elem match
+            case ref: TermRef => i", defined in ${ref.symbol.maybeOwner}"
+            case _ => ""
+          i"""capability ${elem}$levelStr
+              |cannot be included in outer capture set $cs"""
+        else if !elem.tryClassifyAs(cs.classifier) then
+          i"""capability ${elem} is not classified as ${cs.classifier}, therefore it
+              |cannot be included in capture set $cs of ${cs.classifier} elements"""
+        else if cs.isBadRoot(elem) then
+          elem match
+            case elem: FreshCap =>
+              i"""local capability $elem created in ${elem.ccOwner}
+                 |cannot be included in outer capture set $cs"""
+            case _ =>
+              i"universal capability $elem cannot be included in capture set $cs"
+        else
+          i"capability $elem cannot be included in capture set $cs"
+      case _ =>
+        i"capability $elem is not included in capture set $cs"
```

Comment:
nitpick: "is not included" feels a bit awkward, and everywhere else uses "cannot be included".

Reply:
That's intentional. "Cannot be included" is used if the set is a capset variable "Is not included" is it is a constant.

Reply:
I see!